doctor/tasks.py

```
@@ -403,23 +449,33 @@ def extract_from_txt(filepath):
     return content, err, error_code
 
 
-def extract_from_wpd(path):
+def extract_from_wpd(
+    path: str, original_filename: str
+) -> tuple[str, bytes, int]:
     """Extract text from a Word Perfect file
 
     Yes, courts still use these, so we extract their text using wpd2html. Once
     that's done, we pull out the body of the HTML, and do some minor cleanup
     on it.
+
+    :param path: The file path to the Word Perfect (.wpd) file.
+    :param original_filename: The original file name of the Word Perfect (.wpd) file.
+    :return: A tuple containing:
+             - The extracted and cleaned text content (str)
+             - The standard error output from the wpd2html subprocess (bytes)
+             - The return code of the wpd2html subprocess (int). Returns 1 on Python-level errors
```

for failure return codes  we should raise a custom Exception